testsuite/MDAnalysisTests/topology/test_itp.py

```diff
@@ -156,27 +183,58 @@ class TestITPNoMass(ParserBase):
     def universe(self, filename):
         return mda.Universe(filename)
 
+    @pytest.fixture
+    def guessed_masses(self, top):
+        masses = np.array(top.masses.values)
+        names = np.array(top.names.values)
+
+        empty = []
+        for a in masses:
+            empty.append(np.isnan(a))
```

Comment:
```suggestion
        empty = np.isnan(masses)
```

Same as above (and for all copies below)